books/RayTracingInOneWeekend.html

```diff
@@ -1924,7 +1924,7 @@
 
 Some Random Number Utilities
 -----------------------------
-We're going to need need a random number generator that returns real random numbers. This function
+We're going to need a random number generator that returns real random numbers. This function
 should return a canonical random number, which by convention falls in the range $0 ≤ n < 1$. The
 “less than” before the 1 is important, as we will sometimes take advantage of that.
```

Comment:
Please re-wrap this paragraph to 100-character line length; thanks.

Reply:
Done